### PR TITLE
add 'device info' page

### LINKED ISSUE #1910: show Kolibri IP and port in the web UI
During a discussion about native desktop apps and installers, it came up that administrators can often use an easy way to get the current IP and port through the web UI (e.g. Manage -> Device?). 

This would be particularly helpful for users that don't use the terminal, e.g. windows users running the tray application.

see related issue #1595

cc @jtamiace would need design

### PR DESCRIPTION
**Pull request by indirectlylit**
fixes #1910 

image

### CODE REVIEW

**Comment by indirectlylit** [comment]
still to-do: follow up on formatting

File: kolibri/plugins/management/assets/src/device_management/state/mutations/deviceInfoMutations.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export function SET_DEVICE_INFO_PAGE_STATE(state, pageState) {
+  state.pageState = pageState;
```
Comment (rtibbles) [inline kolibri/plugins/management/assets/src/device_management/state/mutations/deviceInfoMutations.js:2]: I still dislike this kind of mutation.

Comment (rtibbles) [reply kolibri/plugins/management/assets/src/device_management/state/mutations/deviceInfoMutations.js:2]: Plus it's identical to all the other page state mutations in the module!

File: kolibri/plugins/management/assets/src/device_management/views/device-info-page/index.vue [hunk 2]
```diff
@@ -0,0 +1,114 @@
+<template>
+
+  <subpage-container>
+    <auth-message v-if="!isSuperuser" authorizedRole="superuser" />
```
Comment (rtibbles) [inline kolibri/plugins/management/assets/src/device_management/views/device-info-page/index.vue]: I had made this comment on the original design: https://github.com/learningequality/kolibri/issues/1910#issuecomment-339171823

File: kolibri/plugins/management/assets/src/device_management/views/device-top-nav.vue [hunk 3]
```diff
@@ -14,6 +15,13 @@
       icon="https"
       :link="linkify(PageNames.MANAGE_PERMISSIONS_PAGE)"
     />
+    <k-navbar-link
+      v-if="isSuperuser"
```
Comment (rtibbles) [inline kolibri/plugins/management/assets/src/device_management/views/device-top-nav.vue]: ibid.

File: kolibri/plugins/management/assets/src/device_management/state/mutations/deviceInfoMutations.js (hunk 1 above)
Comment (indirectlylit) [reply kolibri/plugins/management/assets/src/device_management/state/mutations/deviceInfoMutations.js:2]: yeah .. was just being consistent with the rest of the module

File: kolibri/plugins/management/assets/src/device_management/views/device-info-page/index.vue (hunk 2 above)
Comment (indirectlylit) [reply kolibri/plugins/management/assets/src/device_management/views/device-info-page/index.vue]: correct link? that's jessica's comment

Comment (rtibbles) [reply kolibri/plugins/management/assets/src/device_management/views/device-info-page/index.vue]: I edited her comment :)

Comment (indirectlylit) [reply kolibri/plugins/management/assets/src/device_management/views/device-info-page/index.vue]: oh i see your edit

**Pull request merged**